lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +21,14 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    prepared_text = text.lower()
+    tokens = []
+    for tok in prepared_text:
+        if tok.isalpha():
```

Comment:
try to use continue operator, if token is not alpha

Reply:
done